docs/content/en/docs/Installation/p2p.md

```diff
@@ -14,68 +14,215 @@ Feedback and bug reports are welcome, as we are improving the p2p aspects of Kai
 
 {{% /alert %}}
 
-This section will guide you on how to leverage the peer-to-peer (P2P), full-mesh capabilities of Kairos.
+Deploying Kubernetes at the Edge can be a complex and time-consuming process, especially when it comes to setting up and managing multiple clusters. To make this process easier, Kairos leverages peer-to-peer technology to automatically coordinate and create Kubernetes clusters without the need for a control management interface.
 
-Kairos supports P2P full-mesh out of the box. That allows to seamlessly interconnect clusters and nodes from different regions into a unified overlay network. Additionally, the same network is used for coordinating nodes automatically allowing self-automated, node bootstrap.
+With this feature, users can simply avoid specifying any setting, or just set the number of master nodes in a HA cluster they want to create and the necessary configuration details, and Kairos will take care of the rest. The peer-to-peer technology allows the nodes in the cluster to communicate and coordinate with each other, ensuring that the clusters are set up correctly and efficiently with K3s.
 
-A hybrid network is automatically set up between all the nodes, so there is no need to expose them over the Internet or expose the Kubernetes management API outside, reducing the attacker's exploiting surface.
+This not only saves users time and effort, but also makes it easier to deploy and manage Kubernetes clusters at the Edge, allowing users to focus on running and scaling their applications.
 
-Kairos can be configured to automatically bootstrap a Kubernetes cluster with the full-mesh functionalities, or it can include an additional interface to the machines to let them communicate within a new network segment.
+This feature is currently experimental and can be optionally enabled by adding the following configuration to the node deployment file, if you are not familiar with the installation process, it is suggested to follow the [quickstart](/docs/getting-started):
+
+```yaml
+p2p:
+  # Disabling DHT makes co-ordination to discover nodes only in the local network
+ disable_dht: true #Enabled by default
+ # Automatic cluster deployment configuration
+ auto:
+   ha:
+     # Enables HA controlplane
+     enable: true
+     # number of HA master node (beside the one used for init) for the control-plane
```

Comment:
I think in the code we ended up using this as the total number of nodes. So in this example, there will be 2 master nodes once of which will be used for init. Is that correct?